scribe/lib/scribe/ai/presentation/model/ai_scribe_menu_action.dart

```diff
@@ -0,0 +1,129 @@
+enum AIScribeMenuAction {
+  correctGrammar,
+  improveMakeShorter,
+  improveExpandContext,
+  improveEmojify,
+  improveTransformToBullets,
+  changeToneProfessional,
+  changeToneCasual,
+  changeTonePolite,
+  translateFrench,
+  translateEnglish,
+  translateRussian,
+  translateVietnamese,
+}
+
+extension AIScribeMenuActionExtension on AIScribeMenuAction {
```

Comment:
You don't need to create an extension for enum, you can write a function in enum